readers/common/src/main/java/com/criteo/hadoop/garmadon/reader/GarmadonReader.java

```diff
@@ -270,7 +272,6 @@ synchronized void commitAsync(Map<TopicPartition, OffsetAndMetadata> offsets, Of
             DEFAULT_KAFKA_PROPS.put(ConsumerConfig.ENABLE_AUTO_COMMIT_CONFIG, "false");
             DEFAULT_KAFKA_PROPS.put(ConsumerConfig.KEY_DESERIALIZER_CLASS_CONFIG, StringDeserializer.class.getName());
             DEFAULT_KAFKA_PROPS.put(ConsumerConfig.VALUE_DESERIALIZER_CLASS_CONFIG, ByteArrayDeserializer.class.getName());
-            DEFAULT_KAFKA_PROPS.put(ConsumerConfig.CLIENT_ID_CONFIG, CONSUMER_ID);
```

Comment:
Don't we want to keep a CLIENT_ID_CONFIG as it could help for investigation purrpose?